spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe Api::V1::UsersController do
+
+  describe "editing a user" do
+    it "can add a cohort to a user" do
+      user = create(:user, cohort_id: nil)
+      cohort = create(:cohort)
+
+      put :update, format: :json,
+                   id: user.id,
+                   user: {
+                           cohort_id: cohort.id
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.